packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -45,6 +47,45 @@ interface EnqueueTaskParams {
   name: string;
 }
 
+interface LogFn {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (msg: string, ...args: any[]): void;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  (obj: object, msg?: string, ...args: any[]): void;
+}
+
+/**
+ * A logger standardized logger for Google Cloud Functions
+ */
+export interface GCFLogger {
+  trace: LogFn;
+  debug: LogFn;
+  info: LogFn;
+  warn: LogFn;
+  error: LogFn;
+  metric: LogFn;
+}
+
+export const logger: GCFLogger = initLogger();
+
+export function initLogger(
+  dest?: NodeJS.WritableStream | SonicBoom
+): GCFLogger {
+  const defaultOptions: pino.LoggerOptions = {
+    customLevels: {
+      metric: 30,
```

Comment:
perhaps a comment on this line, what does the 30 represent?

Reply:
30 is the severity level assigned to `info` by GCP and we want `metric` to basically be an alias for `info` at this point. Here's the complete list https://github.com/googleapis/repo-automation-bots/blob/d9f4be19dd844b9cdc69895c4fd583c942e6f6a6/packages/gcf-utils/test/test-helpers.ts#L17-L24